TelCo.ColorCoder/PartialColorPair.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    public partial class ColorPair
```

Comment:
what is the use of partial class here ?

Reply:
removed partial class and changed the logic